src/System.IO.IsolatedStorage/src/project.json

```diff
@@ -8,13 +8,16 @@
         "System.Diagnostics.Tools": "4.3.0-beta-24604-02",
         "System.IO": "4.3.0-beta-24604-02",
         "System.IO.FileSystem": "4.3.0-beta-24604-02",
+        "System.IO.FileSystem.AccessControl": "4.3.0-beta-24604-02",
```

Comment:
A week?

Is it really that bad to do this? I assume we'll hit similar cases eventually as we fill in more functionality in other places.